src/std_lib/time.rs

```diff
@@ -0,0 +1,111 @@
+//! Module containing bindings to rust's `std::time` module.
+//!
+use crate::real_std::{
+    time,
+    convert::TryInto,
+};
+
+use crate::vm::{
+    self,
+    api::{Eff, IO},
+    thread::Thread,
+    types::VmInt,
+    ExternModule,
+};
+
+#[derive(Clone, Debug, Userdata, Trace, VmType)]
+#[gluon(vm_type = "std.time.types.Duration")]
+#[gluon(crate_name = "::vm")]
+#[gluon_trace(skip)]
+struct Duration(time::Duration);
+
+fn as_secs(dur: &Duration) -> VmInt {
+    dur.0.as_secs().try_into().expect("Overflow when trying to convert Duration into seconds")
```

Comment:
Could you return a `RuntimeResult` instead of panicking? (Making gluon panic instead)